opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -227,8 +231,13 @@ public void destroy() {
      * @param httpServletResponse response
      * @return whether request should be traced or not
      */
-    protected boolean isTraced(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
-        return true;
+    protected boolean isNotTraced(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
+        if (skipPattern != null) {
+            String url = httpServletRequest.getRequestURI().substring(httpServletRequest.getContextPath().length());
```

Comment:
Well, I think this might have performance issues, *but*, it's always a good idea to start with the simplest solution that works, and then tweak it *once* you confirm that the performance is not good. A possible future tweak might be to keep a simplistic cache (`Map<String, Boolean>`) locally, with the "raw" URLs you've seen and the decisions that were made. This way, you perform string operations only once per URL, in exchange for an increased memory consumption.